ap/selection/test.py

```diff
@@ -269,6 +276,12 @@ def test(
     # build categories
     self.stack.category_ids(events, config_inst=config_inst, dataset_inst=dataset_inst, **kwargs)
 
+    # for cutflow plots
+    self.stack.process_ids(events, dataset_inst=dataset_inst, **kwargs)
+    # calculating variables before and after applying object cleaning requires two definitions
+    # of the variable; might lead to ambiguities if not careful
+    set_ak_column(events, "ht", self.stack.var_HT(events))
```

Comment:
Mh, why again do you need the variable? Just to make some last cuts later on?

Reply:
Yes, for the simple cutflow plots, no additional variables are required. However, I built the histogram task such that it also creates a variable axis to allow creating ordinary plots as part of the cutflow workflow. But I should probably also define a default when no variable is stored.